lib/dfe_sign_in_api.rb

```diff
@@ -0,0 +1,35 @@
+module DFESignIn
+  class API
+    BASE_URL = DFE_SIGN_IN_URL
+
+    attr_accessor :dfe_sign_in_password
+
+    def initialize
+      self.dfe_sign_in_password = DFE_SIGN_IN_PASSWORD
+    end
+
+    def users(page: 1)
+      token = generate_jwt_token
+      response = HTTParty.get(
+        api_url(page),
+        headers: { 'Authorization' => "Bearer #{token}" }
+      )
+      JSON.parse(response)
+    end
+
+    private
+
+    def api_url(page = 1, records = 25)
```

Comment:
This is a private method, with the records always set to 25.  Is this param useful?

Reply:
It's not useful atm, I forgot to remove, thanks for pointing this out, resolved in  935caaf